week2-6.2-the_2000_run.py

```diff
@@ -0,0 +1,20 @@
+import time
```

Comment:
Notice that you didn't separate the pull requests of both exercises.
Try experimenting later in order to understand branching better.